Build failure: sile, sile.dev sile.doc sile.man on darwin since 0.15.6

## Steps To Reproduce

Steps to reproduce the behavior:

1. build sile on darwin

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log - x86_64-darwin</summary>

```
   Compiling rusile v0.15.5 (/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/rusile)
error: linking with `/nix/store/bhqyl2380k1nhwxw58ki7k9rdgnks1bd-clang-wrapper-16.0.6/bin/cc` failed: exit status: 1
  |
  = note: env -u IPHONEOS_DEPLOYMENT_TARGET -u TVOS_DEPLOYMENT_TARGET -u XROS_DEPLOYMENT_TARGET LC_ALL="C" PATH="/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/bin:/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/bin:/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/bin:/nix/store/maqqdiillgv297234j2z7cc1m0qgx5mz-zstd-1.5.6-bin/bin:/nix/store/09zg7h9pc6m2nqb51qr3js2azcwp1aqw-zstd-1.5.6/bin:/nix/store/7spy1pbif3par1zk0n9y1f8rn2rb67yc-pkg-config-wrapper-0.29.2/bin:/nix/store/g6m0ggpijkm26ygd8vvhyd07sgriz26j-jq-1.7.1-bin/bin:/nix/store/ighhvn60wvid5h3h10fvmsa3qnxw9bnl-cargo-1.82.0/bin:/nix/store/rvq0vrk1m46davhlplbzjii7xprjiiys-rustc-wrapper-1.82.0/bin:/nix/store/bhqyl2380k1nhwxw58ki7k9rdgnks1bd-clang-wrapper-16.0.6/bin:/nix/store/jcxk2xks7gm0am16yw3kzqwc6lm9bq77-clang-16.0.6/bin:/nix/store/w95g6dz0b7byi4w2mgvmwpl692pilqf1-coreutils-9.5/bin:/nix/store/s3pyd5x58drzm0yswmgf00va6qv8sacb-cctools-binutils-darwin-wrapper-1010.6/bin:/nix/store/h8zxz0x9sg80g4zyny8zryjyxb52m19z-cctools-binutils-darwin-1010.6/bin:/nix/store/7sn3hykmkzgngdhjl3apfh1dygrn4n7z-xcbuild-0.1.1-unstable-2019-11-20-xcrun/bin:/nix/store/qv537jaijm8dcw1yi96kjr5l1w7z035q-lua-5.2.4-env/bin:/nix/store/h9kcjw9gjs3yxfa43d2xfiirj72rkrk0-harfbuzz-10.0.1-dev/bin:/nix/store/3n8das6viyfvwfhkkx1z6y7hnfn8ps85-graphite2-1.3.14/bin:/nix/store/hjqpyyhn7ba2fjaq7fh3zv1b1jnqn217-icu4c-74.2-dev/bin:/nix/store/k74yqbslcn1aq6jkvz8raxn1k40d66hv-freetype-2.13.3-dev/bin:/nix/store/azy6wqxqpmi1zis8dbq8f50h0y9rwvs5-bzip2-1.0.8-bin/bin:/nix/store/qkbhbcwj3kg7phqrw3irq09cl51l2s0g-brotli-1.1.0/bin:/nix/store/sq8nhrsl1kbdr75g1hpvrf2akclasxva-libpng-apng-1.6.43-dev/bin:/nix/store/j2vfn35gm6a0ziawmgydphnlrmh9h6a4-fontconfig-2.15.0-bin/bin:/nix/store/ns9fk2cg2k84c2b10cwassgk44gim2c8-libiconv-107/bin:/nix/store/hb2fa5zrq7dskk0hg6f7m6xps5cqvr7j-stylua-0.20.0/bin:/nix/store/gdmaxa67zqa6ad2vnh0kr3jnnwd25qpf-typos-1.27.3/bin:/nix/store/w95g6dz0b7byi4w2mgvmwpl692pilqf1-coreutils-9.5/bin:/nix/store/f0p3d05w8zfjg22jly7p3m0j6p420k3p-findutils-4.10.0/bin:/nix/store/x7lmcc2iycd24q6xga0pkp9ql1b9liqf-diffutils-3.10/bin:/nix/store/x1k6ajjhx3diqw3y661w63hlx378wlgl-gnused-4.9/bin:/nix/store/n2xgwfxfvf2skpxc4vp1vjgmky80bjc3-gnugrep-3.11/bin:/nix/store/36akia2yacfjirqz54il8yz7igzkbza5-gawk-5.3.1/bin:/nix/store/mij3k3il21ra506pca3w1gz5g9d6vc3i-gnutar-1.35/bin:/nix/store/i6w5gh3nn79ym0fhhx0bnqkfp23bg0jq-gzip-1.13/bin:/nix/store/azy6wqxqpmi1zis8dbq8f50h0y9rwvs5-bzip2-1.0.8-bin/bin:/nix/store/wqz4s9ij68jmnp7nphvkh22cbfapmk1w-gnumake-4.4.1/bin:/nix/store/fhngvmgagwwxhak8nzrx5vqbg4g4qiac-bash-5.2p37/bin:/nix/store/9jjpkfywwjhk8qas4sy8vh53rxzsyqz9-patch-2.7.6/bin:/nix/store/mhsmv9fxcmx9jsnhx3iv3mvnvvirbkb2-xz-5.6.3-bin/bin:/nix/store/9l0i2zfsnn2hlbw5nfkhrw19xjsa7xyj-file-5.45/bin" VSLANG="1033" ZERO_AR_DATE="1" "/nix/store/bhqyl2380k1nhwxw58ki7k9rdgnks1bd-clang-wrapper-16.0.6/bin/cc" "-Wl,-exported_symbols_list" "-Wl,/private/tmp/nix-build-sile-0.15.6.drv-0/rustchcbzfR/list" "-arch" "x86_64" "-m64" "/private/tmp/nix-build-sile-0.15.6.drv-0/rustchcbzfR/symbols.o" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/./target/release/deps/rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/./target/release/deps/rusile.rusile.bccd27ce038cd11e-cgu.1.rcgu.o" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/./target/release/deps/rusile.7tdsncn416nmb79vuupj3bl10.rcgu.o" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libsile-1248fd9c84a44daa.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libsemver-a490ed801ca687e4.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libclap-b70fac1d5d1fc626.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libclap_builder-9f00a96ce2d5b443.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libterminal_size-abba9c282c28ba1b.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/librustix-8218d0695b5ce1fb.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libbitflags-dd486ce665ce6b3d.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/liberrno-baf6146df77d6f48.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libstrsim-fee2877e7cc18b83.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libanstream-464cbeee09884cee.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libanstyle_query-251de1947255bc2c.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libis_terminal_polyfill-ea56e8e1d213e05b.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libcolorchoice-c81baaaa76e37ccf.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libanstyle_parse-0f79fe9157ee550a.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libutf8parse-e48768359dc42b96.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libclap_lex-e0a509e29c21b10b.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libanstyle-dfc70de60445f845.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libmlua-ca364491aa9b1beb.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libanyhow-baaae3d29b016be8.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libeither-510f03ab0c116721.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/librustc_hash-0ea67b0ff3efd114.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libparking_lot-fe4416f449f5b549.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libparking_lot_core-6db9ab3dd923f96a.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/liblibc-52f2addc63018d3e.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libcfg_if-da90fdbbf9f970d7.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libsmallvec-143b3dd71fb31f3f.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/liblock_api-297eac42e12174fc.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libscopeguard-2d636dfb854f17be.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libmlua_sys-6d6bcabcd53faafb.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libnum_traits-92ef02381301a408.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libbstr-f88a7393b3bf73fc.rlib" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/target/release/deps/libmemchr-175076c57db67fa3.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libstd-700ae830b7d1ca43.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-8e51590db23eb133.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libobject-601d2586d02aea41.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libmemchr-a4bdfe31916cc14b.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libaddr2line-92dbacab866653c1.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libgimli-6c320ab205f7a69e.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/librustc_demangle-a6b304216560938b.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libstd_detect-fc7c847133b2cb4d.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libhashbrown-ea1caa33b350548d.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_alloc-d6f4057f570a44f8.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libminiz_oxide-8a598cc7b3935cf5.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libadler-e5b3d3a694073492.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libunwind-9040e31bc43b13c2.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libcfg_if-56fc03a939f03a05.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/liblibc-c8b584449bdb9041.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/liballoc-9ef77ec5a2771e74.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_core-268f5aaf08107e43.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libcore-4d884133ae4aa257.rlib" "/nix/store/4q4nkhz63awg9b6xrpahz45n4cyb9wvx-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-df3bf8574a3856a3.rlib" "-liconv" "-lSystem" "-lc" "-lm" "-o" "/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6/./target/release/deps/librusile.dylib" "-Wl,-dead_strip" "-dynamiclib" "-Wl,-dylib" "-nodefaultlibs"
  = note: Undefined symbols for architecture x86_64:
            "_luaL_callmeta", referenced from:
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_luaL_checkstack", referenced from:
                mlua::util::error::init_error_registry::error_tostring::h57365f5265078fa5 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::destructed_error::hbc3b694df33895fa in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua_sys::lua52::compat::lua_rotate::h5caa92db37e4ebf4 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_luaL_error", referenced from:
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_luaL_getmetafield", referenced from:
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_luaL_loadbufferx", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h987b0d61ccfa828f (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hcf7b522efd6f26ac (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
            "_luaL_ref", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h1891eec992b7a6bc (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h6c34ff26810b8a20 (.llvm.4394248584030311217) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.09.rcgu.o)
            "_luaL_traceback", referenced from:
                _luaopen_rusile in rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o
                mlua::util::error::error_traceback::ha4fc07ad8564ad3c in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::error_tostring::h57365f5265078fa5 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::destructed_error::hbc3b694df33895fa in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::state::util::callback_error_ext::habe6ff19a2dc60af in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
            "_lua_absindex", referenced from:
                mlua::state::raw::RawLua::make_userdata_with_metatable::hc4a2b3b18e892d8f in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
                mlua_sys::lua52::compat::lua_rotate::h5caa92db37e4ebf4 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::h23368e3e445719f7 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::h323cea6edf2afab4 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::h43150d65bd172765 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::h9051208862b5bd74 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
                ...
            "_lua_callk", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h987b0d61ccfa828f (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hcf7b522efd6f26ac (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::userdata::init_userdata_metatable::do_call::hb7a73137d0b32d4a in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::init_userdata_metatable::do_call::h5c0617b1b45e3632 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
            "_lua_checkstack", referenced from:
                _luaopen_rusile in rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o
                mlua::util::error::error_traceback::ha4fc07ad8564ad3c in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::h4fbf250ea74fef97 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::error_tostring::h57365f5265078fa5 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::destructed_error::hbc3b694df33895fa in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::state::raw::RawLua::init_from_ptr::h9e30c3bc1af7e49f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::create_table_with_capacity::h77102513d2c9aadb in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                ...
            "_lua_close", referenced from:
                core::ptr::drop_in_place$LT$alloc..rc..RcBox$LT$mlua..types..sync..inner..ReentrantMutex$LT$mlua..state..raw..RawLua$GT$$GT$$GT$::h49d3a496b93c8491 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                _$LT$mlua..state..raw..RawLua$u20$as$u20$core..ops..drop..Drop$GT$::drop::hcc97d7d0d8a75c0b in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                _$LT$alloc..rc..Rc$LT$T$C$A$GT$$u20$as$u20$core..ops..drop..Drop$GT$::drop::ha6fba50926fc0e54 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.12.rcgu.o)
            "_lua_createtable", referenced from:
                mlua::util::error::protect_lua_closure::do_call::hb133e74eb9e5f0db (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::state::raw::RawLua::create_table_with_capacity::h77102513d2c9aadb in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h932fa156c4402d0e (.llvm.4394248584030311217) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.09.rcgu.o)
            "_lua_error", referenced from:
                _luaopen_rusile in rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o
                mlua::util::error::protect_lua_closure::do_call::h987b0d61ccfa828f (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hcf7b522efd6f26ac (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::error_tostring::h57365f5265078fa5 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::destructed_error::hbc3b694df33895fa in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::state::util::callback_error_ext::habe6ff19a2dc60af in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::lua_error_impl::h63be0548ecfbc2d0 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                ...
            "_lua_gc", referenced from:
                mlua::state::Lua::gc_collect::do_call::hd9faaf99c6d2036e (.llvm.5154991789429167657) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.07.rcgu.o)
            "_lua_getallocf", referenced from:
                core::ptr::drop_in_place$LT$alloc..rc..RcBox$LT$mlua..types..sync..inner..ReentrantMutex$LT$mlua..state..raw..RawLua$GT$$GT$$GT$::h49d3a496b93c8491 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                _$LT$mlua..state..raw..RawLua$u20$as$u20$core..ops..drop..Drop$GT$::drop::hcc97d7d0d8a75c0b in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::create_table_with_capacity::h77102513d2c9aadb in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::push_value::h6fd413bb9d9c29a3 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::create_callback::h232c9887c454666e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::Lua::coerce_string::h5c9a44cb6587b72e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.07.rcgu.o)
                _$LT$alloc..rc..Rc$LT$T$C$A$GT$$u20$as$u20$core..ops..drop..Drop$GT$::drop::ha6fba50926fc0e54 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.12.rcgu.o)
                ...
            "_lua_getfield", referenced from:
                mlua::state::raw::RawLua::make_userdata_with_metatable::hc4a2b3b18e892d8f in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
            "_lua_getmetatable", referenced from:
                mlua::state::raw::RawLua::get_userdata_type_id_inner::h05fae93361843131 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::util::userdata::get_internal_userdata::h59b71e628397a409 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::table::Table::has_metatable::h030b38a0f20ee4c4 (.llvm.16053092229523305660) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.01.rcgu.o)
            "_lua_gettop", referenced from:
                _luaopen_rusile in rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o
                mlua::util::error::protect_lua_call::hf1f5e084fe69c6ca in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1255f7c32e8f8fd2 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h162c611f210efb13 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1e949b55752d8c0f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h302224282cd2538d in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h3aa638aba9088049 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                ...
            "_lua_insert", referenced from:
                mlua::state::util::callback_error_ext::habe6ff19a2dc60af in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::util::callback_error_ext::PreallocatedFailure::reserve::hda2dbd9fefc21f2c in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua_sys::lua52::compat::lua_rotate::h5caa92db37e4ebf4 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_lua_isstring", referenced from:
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_lua_newthread", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h1891eec992b7a6bc (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
            "_lua_newuserdata", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h69cdd1e40c9523ce (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h9541c1d4a3424ea8 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hcd9a47be5b3e4581 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::error_tostring::h57365f5265078fa5 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::destructed_error::hbc3b694df33895fa in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::state::util::callback_error_ext::habe6ff19a2dc60af in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::util::callback_error_ext::PreallocatedFailure::reserve::hda2dbd9fefc21f2c in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                ...
            "_lua_pcallk", referenced from:
                mlua::util::error::protect_lua_call::hf1f5e084fe69c6ca in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1255f7c32e8f8fd2 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h162c611f210efb13 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1e949b55752d8c0f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h302224282cd2538d in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h3aa638aba9088049 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h5f05631c3f311a93 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                ...
            "_lua_pushboolean", referenced from:
                mlua::util::error::init_error_registry::h4fbf250ea74fef97 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::state::raw::RawLua::push_value::h6fd413bb9d9c29a3 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::util::userdata::init_internal_metatable::h1c7196cd8178ab9e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::init_internal_metatable::h79a649963bb8b555 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::init_internal_metatable::hb5a753eca5709376 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::init_internal_metatable::he145e2eea20a2652 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::init_userdata_metatable::h3f778e38ffa18974 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                ...
            "_lua_pushcclosure", referenced from:
                mlua::util::error::protect_lua_call::hf1f5e084fe69c6ca in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1255f7c32e8f8fd2 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h162c611f210efb13 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1e949b55752d8c0f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h302224282cd2538d in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h3aa638aba9088049 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h5f05631c3f311a93 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                ...
            "_lua_pushfstring", referenced from:
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_lua_pushinteger", referenced from:
                mlua::state::raw::RawLua::push_value::h6fd413bb9d9c29a3 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::h43150d65bd172765 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::h9051208862b5bd74 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::hda498c2be2f43752 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
            "_lua_pushlightuserdata", referenced from:
                mlua::util::error::protect_lua_closure::h1255f7c32e8f8fd2 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h162c611f210efb13 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1e949b55752d8c0f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h302224282cd2538d in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h3aa638aba9088049 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h5f05631c3f311a93 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h8f177d7ed7dc9814 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                ...
            "_lua_pushlstring", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h773d695969f9c355 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hc5c5785e26b4bf39 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hfc29b1bdecd7b123 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h6586b3a1722367ef (.llvm.4394248584030311217) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.09.rcgu.o)
                mlua::state::raw::RawLua::create_string::h61aa2ea3bfa3b474 in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
                mlua::state::raw::RawLua::create_string::h7821e211869a5d61 in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
                mlua::state::raw::RawLua::make_userdata_with_metatable::hc4a2b3b18e892d8f in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
                ...
            "_lua_pushnil", referenced from:
                _$LT$mlua..types..value_ref..ValueRef$u20$as$u20$core..ops..drop..Drop$GT$::drop::h7c443e6bce1206b1 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::state::raw::RawLua::push_value::h6fd413bb9d9c29a3 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::util::callback_error_ext::PreallocatedFailure::use::h8115ee39dd96558e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::util::callback_error_ext::PreallocatedFailure::release::h2cc5717dd500e94e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::init_userdata_metatable::h3f778e38ffa18974 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::raw::RawLua::make_userdata_with_metatable::hc4a2b3b18e892d8f in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
            "_lua_pushnumber", referenced from:
                mlua::state::raw::RawLua::push_value::h6fd413bb9d9c29a3 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::h43150d65bd172765 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::h9051208862b5bd74 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
                mlua::userdata::registry::UserDataRegistry$LT$T$GT$::box_method::_$u7b$$u7b$closure$u7d$$u7d$::hda498c2be2f43752 (.llvm.6315160915099941883) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.00.rcgu.o)
            "_lua_pushvalue", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h987b0d61ccfa828f (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hcf7b522efd6f26ac (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::h4fbf250ea74fef97 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::rawset_field::h0ccd69a85818a33c in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::state::raw::RawLua::push_value::h6fd413bb9d9c29a3 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::stack_value::h1b9bf19a3e7e8afe in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::push_userdata_metatable::return_field::h557517ebbb6bfdf3 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                ...
            "_lua_rawget", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h773d695969f9c355 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
            "_lua_rawgeti", referenced from:
                mlua::state::raw::RawLua::init_from_ptr::h9e30c3bc1af7e49f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::make_userdata_with_metatable::hc4a2b3b18e892d8f in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
            "_lua_rawgetp", referenced from:
                mlua::util::userdata::get_internal_metatable::h3778d25a9900abab in rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o
                mlua::util::error::init_error_registry::error_tostring::h57365f5265078fa5 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::state::raw::RawLua::init_from_ptr::h9e30c3bc1af7e49f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::util::callback_error_ext::habe6ff19a2dc60af in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::push_internal_userdata::h5134bb569763872e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::push_internal_userdata::h7d7b54677fc79ea8 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::push_internal_userdata::hbe35e73c2660e1ee in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                ...
            "_lua_rawset", referenced from:
                mlua::util::error::protect_lua_closure::do_call::hc5c5785e26b4bf39 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::table::Table::raw_set::do_call::had8cfd28be53f20b in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.00.rcgu.o)
                mlua::table::Table::set::hbbdde98d1a0c3942 in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.01.rcgu.o)
            "_lua_rawsetp", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h3deb007655db6509 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h44452fc7cf7e8ebf (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h789862faed671f53 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h987b0d61ccfa828f (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hcf7b522efd6f26ac (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hf003d0f55a048b23 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::hf4f79fd71947498b (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                ...
            "_lua_remove", referenced from:
                mlua::util::error::protect_lua_call::hf1f5e084fe69c6ca in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1255f7c32e8f8fd2 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h162c611f210efb13 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h1e949b55752d8c0f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h302224282cd2538d in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h3aa638aba9088049 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::h5f05631c3f311a93 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                ...
            "_lua_replace", referenced from:
                _$LT$mlua..types..value_ref..ValueRef$u20$as$u20$core..ops..drop..Drop$GT$::drop::h7c443e6bce1206b1 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::state::util::callback_error_ext::PreallocatedFailure::use::h8115ee39dd96558e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::util::callback_error_ext::PreallocatedFailure::release::h2cc5717dd500e94e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::util::ref_stack_pop::h08afad04e7418d88 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::raw::RawLua::make_userdata_with_metatable::hc4a2b3b18e892d8f in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
                mlua_sys::lua52::compat::lua_rotate::h5caa92db37e4ebf4 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
                ...
            "_lua_setmetatable", referenced from:
                _luaopen_rusile in rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o
                mlua::util::error::init_error_registry::error_tostring::h57365f5265078fa5 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::destructed_error::hbc3b694df33895fa in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::state::util::callback_error_ext::habe6ff19a2dc60af in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::push_internal_userdata::h5134bb569763872e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::push_internal_userdata::h7d7b54677fc79ea8 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::util::userdata::push_internal_userdata::hbe35e73c2660e1ee in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                ...
            "_lua_settable", referenced from:
                mlua::table::Table::set_protected::do_call::hdf555a31fd76d07f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.00.rcgu.o)
            "_lua_settop", referenced from:
                _luaopen_rusile in rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o
                mlua::util::error::pop_error::hccdc6b28eaf0ccbc in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h1891eec992b7a6bc (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h246345e8c0eefcd6 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h3deb007655db6509 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h44452fc7cf7e8ebf (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h69cdd1e40c9523ce (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                ...
            "_lua_toboolean", referenced from:
                mlua::util::to_string::hd2ada0accf1a0f56 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::state::raw::RawLua::stack_value::h1b9bf19a3e7e8afe in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_lua_tolstring", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h246345e8c0eefcd6 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::to_string::hd2ada0accf1a0f56 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::function::Function::wrap_raw::_$u7b$$u7b$closure$u7d$$u7d$::hd275555a12622354 in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.02.rcgu.o)
                mlua::state::Lua::coerce_string::h5c9a44cb6587b72e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.07.rcgu.o)
                mlua::string::String::to_slice::h0eb155888c8e0e4d in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.10.rcgu.o)
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_lua_tonumberx", referenced from:
                mlua::util::to_string::hd2ada0accf1a0f56 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::state::raw::RawLua::stack_value::h1b9bf19a3e7e8afe in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
            "_lua_topointer", referenced from:
                mlua::util::to_string::hd2ada0accf1a0f56 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::state::raw::RawLua::init_from_ptr::h9e30c3bc1af7e49f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::get_userdata_type_id_inner::h05fae93361843131 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::util::userdata::get_internal_userdata::h59b71e628397a409 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::extra::ExtraData::init::h206d1aa7f6e14382 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.04.rcgu.o)
                mlua::state::raw::RawLua::make_userdata_with_metatable::hc4a2b3b18e892d8f in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
                mlua::table::Table::has_metatable::h030b38a0f20ee4c4 (.llvm.16053092229523305660) in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.01.rcgu.o)
                ...
            "_lua_tothread", referenced from:
                mlua::state::raw::RawLua::init_from_ptr::h9e30c3bc1af7e49f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::stack_value::h1b9bf19a3e7e8afe in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
            "_lua_touserdata", referenced from:
                mlua::util::error::protect_lua_closure::do_call::h1891eec992b7a6bc (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h246345e8c0eefcd6 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h3deb007655db6509 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h44452fc7cf7e8ebf (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h69cdd1e40c9523ce (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h773d695969f9c355 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::protect_lua_closure::do_call::h789862faed671f53 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                ...
            "_lua_type", referenced from:
                mlua::util::userdata::get_internal_metatable::h3778d25a9900abab in rusile.rusile.bccd27ce038cd11e-cgu.0.rcgu.o
                mlua::util::error::protect_lua_closure::do_call::h773d695969f9c355 (.llvm.10851299575983546464) in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::error::init_error_registry::error_tostring::h57365f5265078fa5 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.01.rcgu.o)
                mlua::util::to_string::hd2ada0accf1a0f56 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::state::raw::RawLua::init_from_ptr::h9e30c3bc1af7e49f in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::stack_value::h1b9bf19a3e7e8afe in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::util::callback_error_ext::habe6ff19a2dc60af in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                ...
            "_lua_typename", referenced from:
                mlua::util::to_string::hd2ada0accf1a0f56 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.15.rcgu.o)
                mlua::state::raw::RawLua::get_userdata_type_id::h16202276768eed50 in libsile-1248fd9c84a44daa.rlib(sile-1248fd9c84a44daa.sile.fc8ba726b3d62b50-cgu.04.rcgu.o)
                mlua_sys::lua52::compat::luaL_tolstring::h6d757582c691b372 in libmlua_sys-6d6bcabcd53faafb.rlib(mlua_sys-6d6bcabcd53faafb.mlua_sys.b647151f90a13d81-cgu.0.rcgu.o)
            "_lua_xmove", referenced from:
                mlua::state::raw::RawLua::create_table_with_capacity::h77102513d2c9aadb in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::push_value::h6fd413bb9d9c29a3 in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::stack_value::h1b9bf19a3e7e8afe in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::raw::RawLua::create_callback::h232c9887c454666e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.03.rcgu.o)
                mlua::state::util::callback_error_ext::PreallocatedFailure::use::h8115ee39dd96558e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::util::callback_error_ext::PreallocatedFailure::release::h2cc5717dd500e94e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.13.rcgu.o)
                mlua::state::Lua::coerce_string::h5c9a44cb6587b72e in libmlua-ca364491aa9b1beb.rlib(mlua-ca364491aa9b1beb.mlua.89b56ef6b6e8dc5f-cgu.07.rcgu.o)
                ...
          ld: symbol(s) not found for architecture x86_64
          clang-16: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: could not compile `rusile` (lib) due to 1 previous error
install: cannot stat './target/release/librusile.so': No such file or directory
make[2]: *** [Makefile:1946: rusile.so] Error 1
make[2]: Leaving directory '/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6'
make[1]: *** [Makefile:1152: all-recursive] Error 1
make[1]: Leaving directory '/private/tmp/nix-build-sile-0.15.6.drv-0/sile-0.15.6'
make: *** [Makefile:738: all] Error 2
```

</details>

## Additional context

<!-- Add any other context about the problem here. -->

- https://github.com/NixOS/nixpkgs/pull/355895#issuecomment-2510637906
- https://hydra.nixos.org/build/279510156
- Don't set broken flag as this https://github.com/NixOS/nixpkgs/pull/360997
- found in https://github.com/NixOS/nixpkgs/pull/360969#issuecomment-2510607617

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->

## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->
@doronbehar
@alerque

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/de/decasify/package.nix
+++ b/pkgs/by-name/de/decasify/package.nix
@@ -24,8 +24,6 @@ stdenv.mkDerivation (finalAttrs: {
   cargoDeps = rustPlatform.fetchCargoTarball {
     inherit (finalAttrs) pname version src;
     nativeBuildInputs = [ zstd ];
-    # so the cargo fetcher won't try to run the `./configure` script
-    dontConfigure = true;
     hash = "sha256-bD8MYufI87j//7dIAnCzmp4yoOaT81Zv1i7rjWpjPlc=";
   };
 

--- a/pkgs/by-name/gi/git-warp-time/package.nix
+++ b/pkgs/by-name/gi/git-warp-time/package.nix
@@ -27,8 +27,6 @@ stdenv.mkDerivation (finalAttrs: {
   cargoDeps = rustPlatform.fetchCargoTarball {
     inherit (finalAttrs) pname version src;
     nativeBuildInputs = [ zstd ];
-    # so the cargo fetcher won't try to run the `./configure` script
-    dontConfigure = true;
     hash = "sha256-bmClqtH1xU2KOKVbCOrgN14jpLKiA2ZMzWwrOiufwnQ=";
   };
 

--- a/pkgs/by-name/si/sile/package.nix
+++ b/pkgs/by-name/si/sile/package.nix
@@ -167,6 +167,7 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   meta = {
+    broken = stdenv.isDarwin;
     description = "Typesetting system";
     longDescription = ''
       SILE is a typesetting system; its job is to produce beautiful

--- a/pkgs/by-name/si/sile/package.nix
+++ b/pkgs/by-name/si/sile/package.nix
@@ -39,8 +39,6 @@ stdenv.mkDerivation (finalAttrs: {
   cargoDeps = rustPlatform.fetchCargoTarball {
     inherit (finalAttrs) pname version src;
     nativeBuildInputs = [ zstd ];
-    # so the cargo fetcher won't try to run the `./configure` script
-    dontConfigure = true;
     hash = "sha256-iPkXEUC4U1m/ComIDo/J5kwkmM1QdowioNtnSnmMhJ0=";
   };
 

